dataeng/resources/course-exporter-worker.sh

```diff
@@ -0,0 +1,12 @@
+TODAY=$(date +%d)
+
+env | sort
+
+${EXPORTER_VENV}/bin/exporter \
+   --org=${ORG} \
+   --output-bucket=${OUTPUT_BUCKET} \
+   --external-prefix=databases/${DATE:-$TODAY} \
+   --django-admin=${PLATFORM_VENV}/bin/django-admin.py \
+   --django-pythonpath=${PLATFORM_VENV}/edx-platform \
+   --gpg-keys=${GPG_KEYS_PATH} \
```

Comment:
Hmm.   I wonder if this will be actually needed.   One of the ways that the course exporter differs from the regular exporter is that the output is not encrypted.   That is useful for RDX, where we need to actually read and transform the dumps.  And useful for general testing of exporter changes, for example.  (Like the addition we're supposed to do for cohorts, IIRC.)

Oh, wait.   My mistake.   This is actually called by the exporter-slave job, not by the course exporter.  I was misled by the name of this file -- this script is actually exporting an org, not a list of courses.   The output does produce files at the level of individual courses, but the presence of job that takes courses as input made this confusing to me.

Reply:
renamed.